pageserver/src/pgdatadir_mapping.rs

```diff
@@ -755,7 +755,13 @@ impl Timeline {
     ) -> Result<HashMap<String, Bytes>, PageReconstructError> {
         let current_policy = self.last_aux_file_policy.load();
         match current_policy {
-            Some(AuxFilePolicy::V1) | None => self.list_aux_files_v1(lsn, ctx).await,
+            Some(AuxFilePolicy::V1) | None => {
+                warn!(
+                    "tenant={} timeline={} is using deprecated aux file policy V1",
```

Comment:
shouldn't tenant and timeline be already part of the span?

Reply:
https://github.com/neondatabase/neon/pull/8625/commits/cb91592132afa9b31310cce52f99425351726167